src/System.Net.Security/src/System/Net/SecureProtocols/SslStream.cs

```diff
@@ -460,5 +468,15 @@ public override void Write(byte[] buffer, int offset, int count)
         {
             _sslState.SecureStream.Write(buffer, offset, count);
         }
+
+        private static void ValidateSecurityProtocol(SslProtocols protocols)
+        {
+            SslProtocols allowedProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
```

Comment:
Will extract in a const member and place in Common to unify validation code. I'll try to change other places where protocols are filtered.